### PR TITLE
init tik-tak-tok

### PR DESCRIPTION
**Pull request by younghos0811**
ㅅㄷㄴㅅ

### CODE REVIEW

File: hello-reacct/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.<br />
```
Comment (younghos0811) [inline hello-reacct/README.md:9]: npm

**Review by younghos0811 — COMMENTED** [review]
test

File: hello-reacct/README.md (hunk 1 above)
Comment (younghos0811) [reply hello-reacct/README.md:9]: test

File: hello-reacct/.gitignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
```
Comment (younghos0811) [inline hello-reacct/.gitignore:18]: ㅅㄷㄴㅅ

**Comment by Axiw9** [comment]
@8dp
- psa:yo9999[]()

**Pull request closed without merging**